crates/router/src/core/payments/flows/verfiy_flow.rs

```diff
@@ -89,7 +89,17 @@ impl types::VerifyRouterData {
                 )
                 .await
                 .map_err(|err| err.to_verify_failed_response())?;
-                Ok(mandate::mandate_procedure(state, resp, maybe_customer, None).await?)
+
+                let pm_id = super::authorize_flow::save_payment_method(
```

Comment:
Here can the import be qualified as `authorize_flow::save_payment_method`. (Because such changes were request in recent PRs)